src/classes/CRLP_RollupContact_TEST.cls

```diff
@@ -114,6 +119,10 @@ public class CRLP_RollupContact_TEST {
                         CMT_UnitTestData_TEST.RollupRecordType.OppToContact,
                         'npo02__NumberOfClosedOpps__c', CRLP_Operation.RollupType.COUNT, null, 'Fair_Market_Value__c') + ',' +
 
+                CMT_UnitTestData_TEST.createRollupRecord('Sum of Number in multicurrency', null,
```

Comment:
The existing tests do not need to change, and the fields used below do not exist on a packaging org. Creating a new test(s) that fails before the bug fix and then passes after the issue is fixed is enough.